setup.cfg

```diff
@@ -1,3 +1,36 @@
+[tool:pytest]
+addopts = --doctest-modules
```

Comment:
You need to drop this.  You won't get doctests to pass in finite time.

Reply:
Will do.  As mentioned, this was just for curiosity sake.